### PR TITLE
Opencast 2984 nibity attach workflow does not untag google speech captions

### PR DESCRIPTION
**Pull request by npmali**

### CODE REVIEW

File: files/config/default/etc/workflows/uct-nibity-attach-transcripts.xml [hunk 1]
```diff
@@ -10,8 +10,49 @@
 
   <operations>
 
-    <!-- Attach caption/transcript -->
+    <!-- Apply the default workflow configuration -->
+    <operation
+            id="defaults"
+            description="Applying default configuration values">
+      <configurations>
+        <configuration key="googleTranscript">true</configuration>
```
Comment (TurRil) [inline files/config/default/etc/workflows/uct-nibity-attach-transcripts.xml:18]: We are not publishing google so this should be set to **false**

File: files/config/default/etc/workflows/uct-nibity-attach-transcripts.xml [hunk 2]
```diff
@@ -10,8 +10,49 @@
...
+        <configuration key="nibityTranscript">true</configuration>
+        <configuration key="uploadedTranscript">true</configuration>
```
Comment (TurRil) [inline files/config/default/etc/workflows/uct-nibity-attach-transcripts.xml:19]: Same with this one, not publishing upload (**false**)

File: files/config/default/etc/workflows/uct-nibity-attach-transcripts.xml [hunk 3]
```diff
@@ -10,8 +10,49 @@
...
+        <configuration key="download-source-flavors">text/*,captions/*</configuration>
+      </configurations>
+    </operation>
+
+    <!-- Untag Google Speech captions -->
+    <operation
+            id="tag"
+            description="Tagging Google Speech captions for publication">
```
Comment (TurRil) [inline files/config/default/etc/workflows/uct-nibity-attach-transcripts.xml]: Change description to  "Untag Google Speech captions"

File: files/config/default/etc/workflows/uct-nibity-attach-transcripts.xml [hunk 4]
```diff
@@ -10,8 +10,49 @@
...
+            description="Retract published transcripts from event">
+      <configurations>
+        <configuration key="workflow-id">uct-remove-transcripts</configuration>
+      </configurations>
+    </operation>
+
+    <!-- Archive the current state of the mediapackage -->
+    <operation
+            id="snapshot"
+            description="Archiving new assets">
```
Comment (TurRil) [inline files/config/default/etc/workflows/uct-nibity-attach-transcripts.xml]: Description: "Archive the current state"

File: files/config/default/etc/workflows/uct-nibity-attach-transcripts.xml (hunk 1 above)
Comment (npmali) [reply files/config/default/etc/workflows/uct-nibity-attach-transcripts.xml:18]: Theses config settings are for the retract operation, not for publishing, therefore google and uploaded transcripts need to be set to true. 

This workflow attaches a Nibity transcript but before it does that, it removes Google captions and Uploaded captions.

**Review by TurRil — APPROVED** [review]

**Pull request merged**